client/src/globalstyle.js

```diff
@@ -0,0 +1,40 @@
+import { createGlobalStyle } from "styled-components/macro";
+
+const GlobalStyle = createGlobalStyle`
+*,
+*::after,
+*::before {
+    box-sizing: border-box;
+  }
+
+:root {
+    --main-color: #264031;
+    --secondary-color: #778C80;
+    --action-color: #F2CB05;
+    --noaction-color: #fff;
+    --light-font-color: #fff;
+    --dark-font-color: var(--main-color)
+}
+
+body {
+    margin: 0;
+    background: var(--main-color);
+    font-family: "Roboto", sans-serif;
```

Comment:
you can also do this in variables, this mabye helps you in the future in other components ;-)

f.e. --font-body: "Roboto", sans-serif;

Reply:
Good idea! I will do this! 😉